### PR TITLE
Added Teambition 2.0.0.394

### PR DESCRIPTION
**Pull request by VAllens**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20181)

### CODE REVIEW

File: manifests/a/alibaba/teambition/2.0.0.394/alibaba.teambition.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: alibaba.teambition
+PackageVersion: 2.0.0.394
+PackageLocale: en-US
+Publisher: allen
+PackageName: teambition
```
Comment (OfficialEsco) [inline manifests/a/alibaba/teambition/2.0.0.394/alibaba.teambition.locale.en-US.yaml:8]:
```suggestion
Publisher: Teambition
Author: Allen
PackageName: Teambition
```

File: manifests/a/alibaba/teambition/2.0.0.394/alibaba.teambition.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: alibaba.teambition
+PackageVersion: 2.0.0.394
```
Comment (OfficialEsco) [inline manifests/a/alibaba/teambition/2.0.0.394/alibaba.teambition.installer.yaml:5]:
```suggestion
PackageVersion: 2.0.0
```

File: manifests/a/alibaba/teambition/2.0.0.394/alibaba.teambition.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: alibaba.teambition
+PackageVersion: 2.0.0.394
```
Comment (OfficialEsco) [inline manifests/a/alibaba/teambition/2.0.0.394/alibaba.teambition.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.0.0
```

File: manifests/a/alibaba/teambition/2.0.0.394/alibaba.teambition.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: alibaba.teambition
+PackageVersion: 2.0.0.394
```
Comment (OfficialEsco) [inline manifests/a/alibaba/teambition/2.0.0.394/alibaba.teambition.yaml:5]:
```suggestion
PackageVersion: 2.0.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
image
Remember to change the folder name and the file names :)
You can probably change the PackageIdentifier to alibaba.Teambition once you're at it

**Comment by VAllens** [comment]
> image
> Remember to change the folder name and the file names :)
> You can probably change the PackageIdentifier to alibaba.Teambition once you're at it

Thank you for your suggest. :)

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**